packages/common/components/content/blocks/page-header.marko

```diff
@@ -1,11 +1,24 @@
 import { getAsObject } from '@base-cms/object-path';
 
 $ const content = getAsObject(input, 'content');
+$ const specialTypes = ['event', 'webinar', 'whitepaper'];
 
 <endeavor-breadcrumbs-website-section section=content.primarySection />
 <cms-content-name tag="h1" class="page-wrapper__title" obj=input.content />
 <cms-content-teaser tag="p" class="page-wrapper__deck" obj=input.content />
 <if(content.type !== 'contact')>
   <endeavor-content-block-attribution content=input.content />
-  <cms-content-published obj=input.content />
+</if>
+<if(content.type !== 'contact' && !specialTypes.includes(content.type))>
+  <cms-content-published class="page-wrapper__date" obj=input.content />
+</if>
+<if(content.type === 'event')>
+  <p class="page-wrapper__event-date">
+    <endeavor-content-starts obj=input.content/> - <endeavor-content-ends obj=input.content/>
```

Comment:
It looks like this comment is on an earlier commit of the code, rather than on the final commit, as I don't recognize the syntax listed above. The final commit had the "-" and end date wrapped in an if, where they would only show up if an end date existed.

Reply:
Shoot... I had an issue with the code not merging because the branch had sat for a while waiting on submitter approval, and someone else made changes to the file. It looks like when I created a new branch to get around the conflicts that I used an earlier "save" commit of my code, rather than the final code.

Reply:
https://github.com/endeavorb2b/websites/pull/226/files

Will adjust the file to meet the final commit from this one.